examples/Console/TestConsoleExporter.cs

```diff
@@ -29,13 +29,12 @@ internal class TestConsoleExporter
         internal static object Run(ConsoleOptions options)
         {
             // Enable TracerProvider for the source "MyCompany.MyProduct.MyWebServer"
-            // and use a single pipeline with a custom MyProcessor, and Console exporter.
-            using var tracerProvider = Sdk.CreateTracerProvider(
-                (builder) => builder.AddActivitySource("MyCompany.MyProduct.MyWebServer")
-                    .SetResource(Resources.CreateServiceResource("MyServiceName"))
-                    .UseConsoleExporter(
-                        opt => opt.DisplayAsJson = options.DisplayAsJson,
-                        (p) => p.AddProcessor((next) => new MyProcessor(next))));
+            // and use a custom MyProcessor, along with Console exporter.
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder().AddActivitySource("MyCompany.MyProduct.MyWebServer")
+                .SetResource(Resources.CreateServiceResource("MyServiceName"))
+                .AddProcessor(new MyProcessor()) // This must be added before ConsoleExporter
```

Comment:
What happens if it isn't added before ConsoleExporter?

Reply:
Any enrichments done by MyProcessor will not be visible to ConsoleExporter. Spec says processors are called in order they are added.